tools/tsp-client/src/typespec.ts

```diff
@@ -112,6 +114,9 @@ export async function compileTsp({
     emit: [emitterPackage],
     options: overrideOptions,
   };
+  if (trace) {
```

Comment:
Should we not just match and do a pas through

Reply:
You mean have the option be `--trace [<emitter name]` ??

Reply:
We only allow folks to emit with one emitter at a time through this tool so I thought `--trace` is cleaner for folks to pass in, thoughts?

Reply:
If we wanted to enable tracing the compiler what does that config look like? Something like this:
`["@typespec/compiler", "<emitter name>"]`

Reply:
Compiler is not scoped so you’d have to include everything

Reply:
https://typespec.io/docs/handbook/configuration/tracing/

Reply:
Thanks for the info! Updated the code to match what the tsp cli does :)